apps/bmd/src/pages/PollWorkerScreen.test.tsx

```diff
@@ -116,9 +130,265 @@ test('live mode on election day', async () => {
       printer={fakePrinter()}
       tally={getZeroTally(election)}
       togglePollsOpen={jest.fn()}
+      saveTallyToCard={jest.fn()}
+      talliesOnCard={undefined}
+      clearTalliesOnCard={jest.fn()}
     />
   )
 
   // eslint-disable-next-line no-restricted-syntax
   expect(queryByText('Switch to Live Election Mode?')).toBe(null)
 })
+
+test('results combination option is not shown for a non print machine', async () => {
+  const election = electionSampleWithSeal as Election
+  const enableLiveMode = jest.fn()
+  const { queryByText } = render(
+    <PollWorkerScreen
+      activateCardlessBallotStyleId={jest.fn()}
+      appPrecinctId={defaultPrecinctId}
+      ballotsPrintedCount={0}
+      ballotStyleId=""
+      electionDefinition={asElectionDefinition(election)}
+      enableLiveMode={enableLiveMode}
+      hasVotes={false}
+      isLiveMode
+      isPollsOpen={false}
+      machineConfig={fakeMachineConfig({ appMode: VxMarkOnly })}
+      printer={fakePrinter()}
+      tally={getZeroTally(election)}
+      togglePollsOpen={jest.fn()}
+      saveTallyToCard={jest.fn()}
+      talliesOnCard={undefined}
+      clearTalliesOnCard={jest.fn()}
+    />
+  )
+
+  // eslint-disable-next-line no-restricted-syntax
+  expect(queryByText('Combine Results Reports')).toBe(null)
```

Comment:
If you wanted to avoid the eslint disable directive you could do this:
```suggestion
  expect(queryByText('Combine Results Reports')).toBeNull()
```